pkg/operator/deploy/deploy.go

```diff
@@ -190,6 +190,7 @@ func (o *operator) resources() ([]runtime.Object, error) {
 					ReconcilePullSecret:            true,
 					ReconcileRouteFix:              true,
 					ReconcileWorkaroundsController: true,
+					ReconcileMachineSet:            true,
```

Comment:
Wondering if we should set this as false if we deploy in dev mode. I ofthen create a cluster in dev to mess up with the API server and I don't care about worker nodes: I scale to either 1 or 0 worker nodes.

Reply:
If it were up to me I'd leave it `true` by default so that the feature is transparent to the team and others working in dev mode, since there are several ways to work around it if needed. I guess overall I worry about having operator differences in prod and dev because we could be introducing conflicts later.